docs/schema.md

```diff
@@ -567,7 +567,7 @@ The defined ecosystems are:
 | `Debian` | The Debian package ecosystem; the `name` is the name of the source package. The ecosystem string might optionally have a `:<RELEASE>` suffix to scope the package to a particular Debian release. `<RELEASE>` is a numeric version specified in the [Debian distro-info-data](https://debian.pages.debian.net/distro-info-data/debian.csv). For example, the ecosystem string "Debian:7" refers to the Debian 7 (wheezy) release.  |
 | `Alpine` | The Alpine package ecosystem; the `name` is the name of the source package. The ecosystem string must have a `:v<RELEASE-NUMBER>` suffix to scope the package to a particular Alpine release branch (the `v` prefix is required). E.g. `v3.16`. |
 | `Hex` | The package manager for the Erlang ecosystem; the `name` is a Hex package name.  |
-| `Android`  | The Android ecosystem; the `name` field is the Android component name that the patch applies to, as shown in the [Android Security Bulletins](https://source.android.com/security/bulletin) such as `Framework`, `Media Framework` and `Kernel Component`. The exhaustive list of components can be found at the [Appendix](#android-ecosystem-components). |
+| `Android`  | The Android ecosystem; the `name` field is the affected git project name that should be patched, or `:linux_kernel:`. Linux kernel affected package names can be followed by an optional SoC vendor name e.g. `:linux_kernel:Qualcomm`. |
```

Comment:
Is there some documentation around the definitions of "git project name"  and what it means for Android? i.e. expected format, meaning etc. 

Also same for SoCs -- is there a definitive list for these?

Reply:
I have added some clarifying information on Android terminologies hopefully will make it clear what the `name` field means. Also added the list of supported SoC vendors.